src/com/rental/mainController/RentalProcessor.java

```diff
@@ -0,0 +1,41 @@
+package com.rental.mainController;
+
+import com.rental.authorization.Register;
+import com.rental.authorization.Login;
+import com.rental.utils.Communicator;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+public class RentalProcessor {
+    public static void main(String[] args) throws Exception {
+        DocumentBuilderFactory documentBuilderFactory = DocumentBuilderFactory
```

Comment:
Why are these factories used in main? (Of course, this is a rhetorical question). Establishing a connection with file or database in the main loop is at least bad solution.